core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -164,6 +164,14 @@ public static String getMetadataPath(String tablePath) {
     return tablePath + File.separator + METADATA_DIR;
   }
 
+  /**
+   * Return table status file path based on `tablePath`
+   */
+  public static String getTableStatusFilePath(String tablePath) {
+    return getMetadataPath(tablePath) + CarbonCommonConstants.FILE_SEPARATOR
+        + CarbonCommonConstants.LOADMETADATA_FILENAME;
```

Comment:
Actually there is a variable called 'TABLE_STATUS_FILE' in Class 'CarbonTablePath' that represents string ’tablestatus‘, so there is no need to use an extra variable 'LOADMETADATA_FILENAME' in Class ’CarbonCommonConstants‘.  Better to keep only one of them.

Reply:
ok